Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/Solver.cs

```diff
@@ -154,16 +154,13 @@ protected set
 
         #region MonoBehaviour Implementation
 
-        protected virtual void OnValidate()
+        protected virtual void Awake()
```

Comment:
Nit: Now that this happens in Awake (which is way more correct) we could remove the [SerializeField] attribute from SolverHandler since it will be null at serialization/deserialization time.

Reply:
Done!